packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -10,6 +10,8 @@ export interface ISliderState {
 }
 
 const getClassNames = classNamesFunction<ISliderStyleProps, ISliderStyles>();
+export const ONKEYDOWN_TIMEOUT_DURATION = 1000;
```

Comment:
Does this need to be exported as part of the public API or can it be kept local to `Slider`?

Should this be configurable via props?

Usually callbacks like this are debounced at the application layer so it can have fine-grained control on the delay. I defer to component owners tho.